mpmath/libmp/libmpf.py

```diff
@@ -1311,6 +1356,299 @@ def from_str(x, prec=0, rnd=round_fast, base=0):
     return s
 
 
+#----------------------------------------------------------------------------#
+#                              String formatting                             #
+#----------------------------------------------------------------------------#
+
+blog2_10 = 3.3219280948873626
```

Comment:
This constant should be moved up (near math_float_inf, I think) and reused in a couple of places:
```
$ git grep 3.321928 mpmath/libmp/libmpf.py
mpmath/libmp/libmpf.py:    return max(1, int(round(int(n)/3.3219280948873626)-1))
mpmath/libmp/libmpf.py:    return max(1, int(round((int(n)+1)*3.3219280948873626)))
mpmath/libmp/libmpf.py:        prec = int(len(x.as_tuple()[1])*3.3219280948873626)
mpmath/libmp/libmpf.py:        blog2 = 3.3219280948873626
```